packages/ignite-core/test/spec/util/check-module.spec.js

```diff
@@ -9,7 +9,8 @@ const Fs = require("fs");
 const Path = require("path");
 
 describe("check-module", function() {
-  this.timeout(10000);
+  this.timeout(20000);
```

Comment:
timed it locally and npm seems to be really slow

![image](https://user-images.githubusercontent.com/4782871/31417440-4b272cd6-ade4-11e7-9f5d-7af1c10bdaad.png)